### ISSUE TITLE
[BITV] 9.4.1.2/5.2 - The "Activity settings" button has been correctly implemented using the "aria-expanded" attribute, but the accessible name of the button is always the same: aria-label="Open settings menu" (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41901]
which could be confusing for users. The accessible name should be "Activity settings."

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v5-n2
